ESSArch_Core/storage/tests/test_models.py

```diff
@@ -0,0 +1,487 @@
+import os
+import shutil
+import tempfile
+import uuid
+from unittest import TestCase, mock
```

Comment:
`unittest.TestCase` should be replaced with `django.test.TestCase` which resets the database after each test which removes the need of `tearDown`